src/tutorial/counter-app/ratatui-book-tutorial-counter-app/LICENSE

```diff
@@ -0,0 +1,23 @@
+Permission is hereby granted, free of charge, to any
+person obtaining a copy of this software and associated
+documentation files (the "Software"), to deal in the
+Software without restriction, including without
+limitation the rights to use, copy, modify, merge,
+publish, distribute, sublicense, and/or sell copies of
+the Software, and to permit persons to whom the Software
+is furnished to do so, subject to the following
+conditions:
+
+The above copyright notice and this permission notice
+shall be included in all copies or substantial portions
+of the Software.
```

Comment:
The actual copyright notice is missing ;)

Reply:
I used `cargo generate --git https://github.com/ratatui-org/rust-tui-template --name ratatui-book-tutorial-counter-app` for this. I've opened an issue to track it: https://github.com/ratatui-org/rust-tui-template/issues/14

Reply:
Fixed in https://github.com/ratatui-org/ratatui-book/pull/31/commits/807f94ba880546a78e51309c0206b76d3d34dbc6